indico/modules/events/blueprint.py

```diff
@@ -25,6 +27,11 @@
 _bp.add_url_rule('/admin/event-labels/', 'event_labels', RHEventLabels, methods=('GET', 'POST'))
 _bp.add_url_rule('/admin/event-labels/create', 'create_event_label', RHCreateEventLabel, methods=('GET', 'POST'))
 _bp.add_url_rule('/admin/unlisted-events/', 'unlisted_events', RHUnlistedEvents, methods=('GET', 'POST'))
+_bp.add_url_rule('/admin/autolinker/', 'autolinker_admin', RHAutoLinker)
+_bp.add_url_rule('/admin/autolinker/config', 'autolinker_config', RHAutoLinkerConfig, methods=('POST',))
+
+# Auto-linker rules
+_bp.add_url_rule('/admin/autolinker/rules', 'autolinker_rules', RHAutoLinkerRules)
```

Comment:
I don't think this should be under `/admin/` since it's a semi-public endpoint, Maybe `/api/autolinker-rules` instead?